src/libs/thorvg/tvgStr.cpp

```diff
@@ -186,7 +186,7 @@ float strToFloat(const char *nPtr, char **endPtr)
         auto scale = 1.0f;
 
         while (exponentPart >= 8U) {
-            scale *= 1E8;
+            scale *= (float)1E8;
```

Comment:
Following the style in this file.
```
        auto scale = 1.0f;
```
Let's add `f` suffix instead.

```suggestion
            scale *= 1E8f;
```

Reply:
done